src/vs/workbench/contrib/debug/browser/debugCommands.ts

```diff
@@ -94,9 +95,11 @@ export const CALLSTACK_UP_LABEL = { value: nls.localize('callStackUp', "Navigate
 export const CALLSTACK_DOWN_LABEL = { value: nls.localize('callStackDown', "Navigate Down Call Stack"), original: 'Navigate Down Call Stack' };
 
 export const SELECT_DEBUG_CONSOLE_LABEL = { value: nls.localize('selectDebugConsole', "Select Debug Console"), original: 'Select Debug Console' };
+export const SELECT_DEBUG_SESSION_LABEL = { value: nls.localize('selectDebugSession', "Select Debug Session"), original: 'Select Debug Session' };
 
 export const DEBUG_QUICK_ACCESS_PREFIX = 'debug ';
 export const DEBUG_CONSOLE_QUICK_ACCESS_PREFIX = 'debug consoles ';
+export const DEBUG_SESSION_QUICK_ACCESS_PREFIX = 'debug sessions ';
```

Comment:
Can't remember whether we talked about this already, but when I discussed this with Tyler, he was thinking that since we don't have strong guidelines on what belongs in a quick access, we won't go adding new ones yet. We will revisit it later this year. So for now this PR should add a command to the command palette to select a debug session, but not the quick access.